specs/core/0_beacon-chain.md

```diff
@@ -465,6 +471,16 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
     'persistent_committees': [['uint24']],
     'persistent_committee_reassignments': [ShardReassignmentRecord],
 
+    # Proof of custody
+    # Placeholders for now; ProofOfCustodyChallenge is defined in phase 1, implementers can
+    # put a dummy class in for now, as the list will remain empty throughout phase 0
+    'poc_challenges': [ProofOfCustodyChallenge],
+    # Proof of custody commitment
```

Comment:
the following three fields belong in the `ValidatorRecord`

* `poc_commitment`
* `last_poc_change_slot`
* `second_last_poc_change_slot`